### ISSUE TITLE
Expo support

### ISSUE THREAD

**Issue by BrodaNoel** [nesterapp/react-native-streetview#52]
Hello!

Does this library supports Expo?

**Comment by amitpdev** [comment]
Yes. It works with Expo development builds and projects using expo prebuild. It’s a native module, so it doesn’t run in Expo Go. After installing the package, set up your Google Maps API keys the same way as on a bare React Native app (iOS AppDelegate / Android manifest, or your usual Expo config for native keys). Details are in the README under Expo.
